package.json

```diff
@@ -24,19 +24,18 @@
     "deploy": "gh-pages -d example/build"
   },
   "peerDependencies": {
-    "react": "^16.0.0"
+    "react": "^17.0.2"
```

Comment:
[q] can we make this 16 or 17? I don't think we want to mandate clients are running v17 to use us.

Looking at Stripe for an example https://github.com/stripe/react-stripe-js/pull/147/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L109

Reply:
Oh nice! Didn't realize we can `or` these